Formula/twine-pypi.rb

```diff
@@ -0,0 +1,45 @@
+class TwinePypi < Formula
+  include Language::Python::Virtualenv
+
+  desc "Utilities for interacting with PyPI"
+  homepage "https://pypi.python.org"
+  url "https://github.com/pypa/twine/archive/1.8.1.tar.gz"
+  sha256 "918d7ebab52280bac573af9a6c80622202f0cdeab14cb3360f9ffb9848be946d"
+  head "https://github.com/pypa/twine.git"
+
+  depends_on :python
```

Comment:
Yeah, that seems fine actually. I've always wondered if the `=> :recommended` was to give people a chance to use a custom Python instead of having Homebrew's installed whenever a bottle was poured, but that's not exactly a problem for < 10.6 these days, to say the least.